flow360/component/simulation/meshing_param/edge_params.py

```diff
@@ -0,0 +1,48 @@
+from typing import List, Literal, Optional, Union
+
+import pydantic as pd
+
+from flow360.component.simulation.framework.base_model import Flow360BaseModel
+from flow360.component.simulation.framework.entity_base import EntityList
+from flow360.component.simulation.primitives import Edge
+
+
+class ByAngle(Flow360BaseModel):
+    """Surface edge refinement by specifying curvature resolution in degrees"""
+
+    type: Literal["angle"] = pd.Field("angle", frozen=True)
+    value: u.degree = pd.Field()  # This should have dimension of angle
+
+
+class ByHeight(Flow360BaseModel):
+    """Surface edge refinement by specifying first layer height of the anisotropic layers"""
+
+    type: Literal["height"] = pd.Field("height", frozen=True)
+    value: LengthType.PositiveFloat = pd.Field()
+
+
+class ByAspectRatio(Flow360BaseModel):
```

Comment:
This should be renamed as AspectRatioBasedRefinement.

ByAspectRatio is not good english and neither is a good class name. similar for the others.

If the above is too long, just AspectRatio is better